src/NServiceBus.Metrics/Reporters/NServiceBusMetricReport.cs

```diff
@@ -16,13 +16,13 @@
 
 class NServiceBusMetricReport : MetricsReport
 {
-    string destination;
+    UnicastAddressTag destination;
     IDispatchMessages dispatcher;
 
     public NServiceBusMetricReport(IDispatchMessages dispatcher, string destination)
     {
         this.dispatcher = dispatcher;
-        this.destination = destination;
+        this.destination = new UnicastAddressTag(destination);
```

Comment:
UnicastTag is low level core seam, usually we use string as destination exposed to the user. This change is just so that I don't have to create it over and over again since we always send to the same destination